ui/src/main/webapp/index.html

```diff
@@ -67,7 +70,7 @@
             packageConfigPaths: ['node_modules/*/package.json']
         });
     </script>
-    <script src="node_modules/rxjs/bundles/Rx.umd.js"></script>
+    <script src="node_modules/rxjs/bundles/Rx.min.js"></script>
```

Comment:
From what I saw, UMD is a package containing all deps. Are the other deps included in the deps configured in the project?